### ISSUE TITLE
Can't exit from full screen mode enabled in "Sequence Viewer" or "Media Viewer"

### ISSUE THREAD

**Issue by Symbian9** [olive-editor/olive#330]
### Actual behavior

After switch to full screen mode from "Sequence Viewer" or "Media Viewer", there are NO way leave it and no any buttons for close full screen window (if just not kill full screen window by global shortcut: `Alt`+`F4`).

### Expected behavior

There should be "Escape Full Screen" button (that could automatically hide if mouse cursor not moved more than few seconds) in full screen mode for "Sequence Viewer" or "Media Viewer".

Also, click `Esc` (`Escape`) on keyboard should escape full screen.

**Comment by itsmattkc** [comment]
Right click Viewer > Show Fullscreen > Disable is the way to close, but yes there should probably be more ways

**Comment by Symbian9** [comment]
> Right click Viewer > Show Fullscreen > Disable is the way to close

It not work for me. `Olive-5776387-x86_64.AppImage`

**Comment by Symbian9** [comment]
> Fullscreen can now be quit with Esc or an onscreen button in `a042520`

Yes it work, but *onscreen button* shown ugly on older PC (with OpenGL1.x only?)

![pic.1](https://i.imgur.com/UsvsWE3.png)

Is it possible fix it to?

**Comment by itsmattkc** [comment]
Yeah the right click I was referring to was on the Viewer panel, but if it's fullscreen with only one monitor that doesn't help.

I'm not sure what can be done regarding the broken text. I'm using Qt's painting functions and they're standardized for OpenGL ES 2.0+, so if your GPU doesn't full support that standard, that probably leads to the effect you see.
